maker/src/orderbook_ws.rs

```diff
@@ -79,6 +86,7 @@ impl Client {
                 let authenticate = auth_fn;
                 match orderbook_client::subscribe_with_authentication(url, authenticate).await {
                     Ok((mut sink, mut stream)) => {
+                        let _ = orderbook_status.send(ServiceStatus::Online);
```

Comment:
:upside_down_face: We could also send this once we are authenticated. There's a relevant message handled below.

Reply:
good idea! it looks perhaps a bit more convoluted now, but it's certainly more correct. :-)